docs/manual_pl.md

```diff
@@ -0,0 +1,388 @@
+<<<<<<< Updated upstream
```

Comment:
Here, a merge conflict needs to be resolved. When automatically combining changes from different sources (e.g. rebasing, merging,...) git could not decide which (if any) of the two versions is the correct one.
Changes from one source are between <<<<< and =====, from the other between ===== and >>>>>.

You have to resolve these manually.